### ISSUE TITLE
Specify the scroll element rather than using the window object

### ISSUE THREAD

**Issue by xialvjun** [sergiodxa/react-lazy-image#2]

**Comment by xialvjun** [comment]
Now I use 

```
<ScrollElement onScroll={()=>Object.keys(this.lazyimgs).forEach(key => this.lazyimgs[key].checkViewport())}>
...
{listitem.map(item => (
<Image key={item.id} ref={e=>this.lazyimgs[item.id]=e} source={item.picture_url} style={{display:'block'}} />
))}
...
</ScrollElement>
```

...The ScrollElement of course is not window, and nor the parent of Image.

**Comment by sergiodxa** [comment]
Ok, this is a good idea, I'm going to implement it 👍

**Comment by sergiodxa** [comment]
Closed in favor of #14
